exercises/anagram/cases_test.go

```diff
@@ -20,23 +20,6 @@ var testCases = []struct {
 			"pants"},
 		expected: []string{},
 	},
-	{
-		description: "detects simple anagram",
-		subject:     "ant",
-		candidates: []string{
-			"tan",
-			"stand",
-			"at"},
-		expected: []string{
-			"tan"},
-	},
-	{
-		description: "does not detect false positives",
-		subject:     "galea",
-		candidates: []string{
-			"eagle"},
-		expected: []string{},
-	},
```

Comment:
When a generator update causes test cases to drop out, it is probably best to use a separate PR for that one exercise, and note in the commit log that test cases changed.  But not a problem. Just a note for future practice.

Reply:
ok! Thanks for pointing it out :)